webapp/src/styles/main.scss

```diff
@@ -240,3 +240,123 @@ h1 {
         width: 126px;
     }
 }
+
+// Tooltip arrow width
```

Comment:
Great work here!

Can we move this to a separate file to keeps things better structured and managable?

Reply:
Will do :+1: